### PR TITLE
Miscellaneous minor improvements

### PR DESCRIPTION
**Pull request by tjanez**
Improving robustness of `goea.py` tool, debuggability of `Feature` and `Mapping` models and ensuring no tests could ever connect to `qa2.genialis.com` server.

### CODE REVIEW

File: resolwe_bio/tools/goea.py [hunk 1]
```diff
@@ -33,27 +34,36 @@ def main():
...
         target_ids = genes
     else:
-        features = res.mapping.filter(
+        mapping_res = res.mapping.filter(
             source_db=args.source_db,
             source_species=args.species,
             target_db=args.target_db,
             target_species=args.species,
-            source_id=genes)
+            source_id=genes
```
Comment (dblenkus) [inline resolwe_bio/tools/goea.py]: Add a trailing comma.

File: resolwe_bio/tools/goea.py [hunk 2]
```diff
@@ -33,27 +34,36 @@ def main():
...
             exit(1)
 
-        target_ids = [str(feature.target_id) for feature in features]
+        mappings = {}
+        for m in mapping_res:
+            if m.source_id in genes:
+                if m.source_id not in mappings:
+                    mappings[m.source_id] = m.target_id
+                else:
+                    print(warning("Mapping {} returned multiple times.".format(m)))
```
Comment (dblenkus) [inline resolwe_bio/tools/goea.py]: Add a single quotes around `{}`.

Comment (tjanez) [reply resolwe_bio/tools/goea.py]: Do you think it is necessary?

Here is an example output of the current code:
```
Mapping <Mapping source_db='NCBI' source_id='56535' target_db='MGI' target_id='MGI:1929646'> returned multiple times.
```

File: resolwe_bio/tools/goea.py (hunk 1 above)
Comment (tjanez) [reply resolwe_bio/tools/goea.py]: Done.

File: resolwe_bio/tools/goea.py (hunk 2 above)
Comment (dblenkus) [reply resolwe_bio/tools/goea.py]: Ok, leave it like this.

**Pull request merged**